scripts/gen_syscall_header.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+def gen_macro(ret, argc, inline=False):
+    sys.stdout.write("K_SYSCALL_DECLARE%d%s%s(id, name" % (argc,
+        ("" if ret else "_VOID"), ("_INLINE" if inline else "")))
```

Comment:
Not a python pedant, but if you're using "sys.stdout.write" this pervasively to generate human readable ASCII, that seems like exactly what "print" is intended for.

Reply:
print has an implicit newline which gets in the way